### PR TITLE
Upgrading USFM-GRAMMAR to 3.0.0b2

### PR DESCRIPTION
**Pull request by AthulyaMS**
#427 
1 ) Upgarding USFM-GRAMMAR to 3.0.0b2
2 )Remove the JSON of Bibles from DB

### CODE REVIEW

**Comment by AthulyaMS** [comment]
#427

**Review by Joel-Joseph-George — CHANGES_REQUESTED** [review]
Hey athulya, you are trying to merge it into master, Could you change it to version-2

File: app/crud/files_crud.py [hunk 1]
```diff
@@ -5,20 +5,24 @@
...
+        for item in converted_content['book']:
+            if item != 'chapters':
+                output_content['book'][item] = converted_content['book'][item]
+            else:
+                output_content['book']['chapters'] = []
+                for chapter_dict in converted_content['book']['chapters']:
+                    if int(chapter_dict.get('chapterNumber', 0)) == chapter:
+                        output_content['book']['chapters'].append(chapter_dict)
+                        break
+
```
Comment (kavitharaju) [inline app/crud/files_crud.py:22]: Is this code working for the new output you get???

File: app/crud/files_crud.py [hunk 2]
```diff
@@ -56,15 +60,20 @@ def extract_usx_chapter(converted_content, chapter:int):
...
         case usfm_grammar.Format.JSON:
-            output_content = usfm_parser.to_dict(content_filter)
-            if chapter is not None:
-                output_content = extract_dict_chapter(output_content, chapter)
+            if content_filter == usfm_grammar.Filter.PARAGRAPHS:
+                output_content = usfm_parser.to_usj()
+                if chapter is not None:
+                    output_content = extract_usj_chapter(output_content, chapter)
+            elif content_filter == usfm_grammar.Filter.SCRIPTURE_PARAGRAPHS:
+                output_content = usfm_parser.to_usj(scripture=True)
```
Comment (kavitharaju) [inline app/crud/files_crud.py:74]: scripture=True? We dont have such an option in `to_usj()` function.

**Review by kavitharaju — COMMENTED** [review]
You can refer this notebook also to see how to use the tool and how the output structure look like
https://github.com/Bridgeconn/usfm-grammar/blob/version-3/docs/API%20guide%20for%20python%20usfm_grammar.ipynb

File: app/test/test_file_ops.py [hunk 3]
```diff
@@ -21,72 +20,60 @@ def test_usfm_to_json():
...
 
-    resp = client.put(f"{UNIT_URL}usfm/to/json?chapter=2",
-        json=gospel_books_data[0])
-    output = resp.json()
-    assert "book" in output
-    assert "chapters" in output['book']
-    assert len(output['book']['chapters']) == 1
-    assert int(output['book']['chapters'][0]['chapterNumber']) == 2
+    # Check if 'book' has 'chapters' key and 'chapters' is an empty list for this particular chapter filter
+    if 'chapters' in output['book']:
```
Comment (kavitharaju) [inline app/test/test_file_ops.py:56]: This condition will never be true as per the new structure

File: app/test/test_file_ops.py [hunk 4]
```diff
@@ -21,72 +20,60 @@ def test_usfm_to_json():
...
+        assert len(output['book']['chapters']) == 0
+    else:
+        
+        assert "book" in output
+        assert "chapters" not in output['book']
```
Comment (kavitharaju) [inline app/test/test_file_ops.py:61]: I wonder how this condition would work either.... :thinking:

**Pull request merged**